app/views/layouts/application.html.erb

```diff
@@ -4,15 +4,16 @@
     <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
     <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
     <%= csrf_meta_tags %>
-    <title>Hogwarts
+    <title>Hogwarts</title>
   </head>
   <body>
+    <h1>Welcome to Hogwarts!</h1>
     <nav>
-      <h1>Welcome to Hogwarts!</h1>
-      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", student_path %> | <%= link_to "Enroll", student_path %></h2>
+
+       <h2> <%= link_to "View Houses", houses_index_path %> | <%= link_to "View Students", students_path %> | <%= link_to "Enroll", students_path %></h2>
```

Comment:
Do you see `houses_index` in the `rails routes` output? My guess is that the proper path if `houses_path`.